vendor/github.com/openshift/api/config/v1/types_feature.go

```diff
@@ -97,44 +97,99 @@ type FeatureGateEnabledDisabled struct {
 //
 // If you put an item in either of these lists, put your area and name on it so we can find owners.
 var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
-	Default: {
-		Enabled: []string{
-			"RotateKubeletServerCertificate", // sig-pod, sjenning
-			"SupportPodPidsLimit",            // sig-pod, sjenning
-			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
-			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
-			"SCTPSupport",                    // sig-network, ccallend
-		},
-		Disabled: []string{
-			"LegacyNodeRoleBehavior", // sig-scheduling, ccoleman
-		},
-	},
+	Default: defaultFeatures,
 	CustomNoUpgrade: {
 		Enabled:  []string{},
 		Disabled: []string{},
 	},
-	TechPreviewNoUpgrade: {
-		Enabled: []string{
-			"RotateKubeletServerCertificate", // sig-pod, sjenning
-			"SupportPodPidsLimit",            // sig-pod, sjenning
-			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
-			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
-		},
-		Disabled: []string{
-			"LegacyNodeRoleBehavior", // sig-scheduling, ccoleman
-		},
+	TechPreviewNoUpgrade: newDefaultFeatures().toFeatures(),
+	LatencySensitive: newDefaultFeatures().
+		with(
+			"TopologyManager", // sig-pod, sjenning
+		).
+		toFeatures(),
+}
+
+var defaultFeatures = &FeatureGateEnabledDisabled{
+	Enabled: []string{
+		"RotateKubeletServerCertificate", // sig-pod, sjenning
+		"SupportPodPidsLimit",            // sig-pod, sjenning
+		"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
+		"ServiceNodeExclusion",           // sig-scheduling, ccoleman
+		"SCTPSupport",                    // sig-network, ccallend
+		"IPv6DualStack",                  // sig-network, ccoleman
```

Comment:
this causes `ci/prow/unit` to fail

```
--- FAIL: TestRenderCommand (0.10s)
    render_test.go:295: "scenario 1 checks feature gates" reports incorrect config file, error: expected to get exactly 7 feature gates but found 6: expected=[RotateKubeletServerCertificate=true SupportPodPidsLimit=true NodeDisruptionExclusion=true ServiceNodeExclusion=true SCTPSupport=true IPv6DualStack=true LegacyNodeRoleBehavior=false] got=[RotateKubeletServerCertificate=true SupportPodPidsLimit=true NodeDisruptionExclusion=true ServiceNodeExclusion=true SCTPSupport=true LegacyNodeRoleBehavior=false]
```

Reply:
@sttts does it mean we have to enable that feature in `cluster-kube-apiserver-operator` ?

Reply:
it seems to be coming from https://github.com/openshift/cluster-kube-apiserver-operator/blob/88c1f9f76d0078f8dc5580d64ceed2852f5f36ed/bindata/bootkube/config/bootstrap-config-overrides.yaml#L59

Reply:
yeah, just update that list